packages/bridge-ui-v2/src/components/Bridge/Amount.svelte

```diff
@@ -33,17 +37,19 @@
   let inputBox: InputBox;
   let computingMaxAmount = false;
 
+  onMount(() => {
+    clearAmount();
```

Comment:
Why do we want to clear on mount? If the reason is to clear the state, before we actually wanted to leave the amount on purpose when switching pages

Reply:
It is not persisting (visibly) right now. We either clear it here or we make sure it is also displayed.